package.json

```diff
@@ -12,7 +12,8 @@
     "clean": "./node_modules/.bin/lerna run clean --stream",
     "build": "./node_modules/.bin/lerna run build --stream",
     "serve": "./node_modules/.bin/http-server -a 127.0.0.1 -o /packages/modelviewer.dev -c-1",
-    "update:package-lock": "rm -f ./package-lock.json && npm i && ./node_modules/.bin/lerna run update:package-lock --stream"
+    "update:package-lock": "rm -f ./package-lock.json && npm i && ./node_modules/.bin/lerna run update:package-lock --stream",
```

Comment:
I have no problem with removing the shared-assets package-lock, but we should also remove its package.json.

Reply:
I think we still need the package.json for the script

Reply:
It seems to work, so we can skip it for now, but let's do a follow-on to clean up shared-assets.

Reply:
Yeah, when we run "npm install", it does call "prepare" script [aligns with npm documentation] which ends up fetching the khornos assets. And I agree that's outside of the scope of this task